### ISSUE TITLE
♻️ Upgrade hydrogen on `tc-report`

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#4164]
## ♻️ Refactor

The report is sitting on an old version of Hydrogen that took a library approach.

## 🙋‍♀️ Proposed Solution

- upgrade Hydrogen to the new on-demand compiler version
- update data-h2 attributes accordingly
- rebuild static site

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the debt was addressed and expected functionality has not been affected.

- [ ] Hydrogen library uses latest version
- [ ] tc-report works and looks the same as before

**Comment by gobyrne** [comment]
@JoshBeveridge thoughts?

**Comment by JoshBeveridge** [comment]
Yeah, Matt and I talked about this. It's not a priority, but I'll upgrade it if I ever have downtime. 

If the language breakdown is an issue for us, I posted a way to ignore directories from the calculation in Slack, using the `.gitattributes` file.

**Comment by mnigh** [comment]
> If the language breakdown is an issue for us, I posted a way to ignore directories from the calculation in Slack, using the `.gitattributes` file.

I've created a separate issue for this since it is a much smaller task:

- https://github.com/GCTC-NTGC/gc-digital-talent/issues/5751

**Comment by vd1992** [comment]
This still something on the radar or is it 🧊

**Comment by substrae** [comment]
It's still buried in my todo list, but it would be nice to consolidate the whole thing.

**Comment by tristan-orourke** [comment]
Moving this to Icebox instead of closing it so that @substrae can see still himself assigned to it. ;) Feel free to close this @substrae if you ever decide to give up on it.
